doc/02-component-proof-of-concept/02-component-proof-of-concept.md

```diff
@@ -114,8 +106,7 @@ Below is further rationale/explanation for the rubric items above:
 > to create a new design. In you do end up picking one at random, you should
 > disclose that here as well.
 
-<!-- TODO: briefly argue your choice of design or design something
-new; then delete this comment -->
+For this assignment, I'll choose to builld upon the GameStatsTracker component. Some reasons I chose this one was because I really like to play videogames so there's personal interest and engagement in it. It could also be useful in the real world, as a lot of gamers like to track their stats and performance. It also demonstrates a variety of methods that add data, retrieve stats, and do calculations. Lastly, it's also the one that may be most achievable with the time constraints.
 
 > Once you've argued your choice of design, make a branch in your new repo called
```

Comment:
GitHub clearly has a bug because your paragraph isn't showing here, but it is showing here: https://github.com/drowe29/component-project/blob/proof-of-concept/doc/02-component-proof-of-concept/02-component-proof-of-concept.md